5-WebApp-AuthZ/5-2-Groups/AppCreationScripts/sample.json

```diff
@@ -5,9 +5,23 @@
     "Client": "ASP.NET Core Web App",
     "Service": "Microsoft Graph",
     "RepositoryUrl": "microsoft-identity-platform-aspnetcore-webapp-tutorial",
-    "Endpoint": "AAD v2.0"
+    "Endpoint": "AAD v2.0",
+    "platform": "AspNetCore",
+    "Languages": [
+      "csharp"
+    ],
+    "Products": [
+      "aspnet-core",
+      "azure-active-directory"
```

Comment:
```suggestion
      "azure-active-directory"
      "microsoft-identity-web"
```

Reply:
`microsoft-identity-web` as it appears here: https://review.learn.microsoft.com/en-us/help/platform/metadata-taxonomies?branch=main